sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DeploymentStep.java

```diff
@@ -136,6 +136,7 @@ public synchronized void update(Protos.TaskStatus status) {
             case TASK_FAILED:
             case TASK_KILLED:
             case TASK_KILLING:
+            case TASK_LOST:
```

Comment:
It would be good to add a unit tests to prevent regressions here. Also, considering that this is an error, would should the log level of the `default` case not be increased?